src/Maker/MakeRegistrationForm.php

```diff
@@ -138,9 +139,16 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
 
         $input->setArgument('will-verify-email', $willVerify);
 
+        // This must be preset to true to avoid code being generated if $willVerify === false
+        $input->setArgument('verify-email-with-auth', true);
+
         if ($willVerify) {
             $this->checkComponentsExist($io);
 
+            $verifyWithAuth = $io->confirm('Do you want to require the user to be authenticated to verify their email?', true);
+
+            $input->setArgument('verify-email-with-auth', $verifyWithAuth);
```

Comment:
we can combine these into 1 line and avoid the `$verifyWithAuth` argument.

Reply:
done